main.tf

```diff
@@ -1,15 +1,91 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  enabled                   = module.this.enabled
+  prometheus_policy_enabled = local.enabled && var.prometheus_policy_enabled
+}
+
+resource "aws_grafana_workspace" "this" {
+  count = local.enabled ? 1 : 0
+
+  name        = module.this.id
+  description = "Amazon Managed Grafana for ${module.this.id}"
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+  account_access_type      = "CURRENT_ACCOUNT"
```

Comment:
This shouldn't be a parameter?

Reply:
the alternate access type, `ORGANIZATION`, requires additional configuration that I haven't tested. It's probably simple, but it should be a feature request